tests/lib/path.py

```diff
@@ -74,7 +74,7 @@ def __repr__(self):
         return "Path({inner})".format(inner=str.__repr__(self))
 
     def __hash__(self):
-        return str.__hash__(self)
+        return super().__hash__()
```

Comment:
I’m curious why this is even needed.

Reply:
Good catch. This is not necessary. I've removed it in the newly added commit.